pages/common/airpaste.md

```diff
@@ -0,0 +1,15 @@
+# airpaste
+
+> Share messages and files locally.
+
+- Send file:
+
+`airpaste < file.txt`
+
+- Receive file:
+
+`airpaste > file.txt`
+
+- Create/Join channel:
+
+`airpaste 🏀`
```

Comment:
Is that an emoji ? Is there a way to get that from keyboard ? Or do we need to copy-paste always ?

Reply:
It's just an example, I thought it was fun, _airpasting a basketball_ 🤠

I could be anything e.g. `airpaste channelOne`

`Command + Control + Spacebar` on macOS for emoji input

Reply:
Let's change it to `airpaste {{channel_name}}` for clarity.

Reply:
I'd agree with changing it to a token, emojis are often unlikely to display on Windows.

![Windows Example](https://image.prntscr.com/image/htO9ca9FRCCp1ABnrzxGmQ.png)